addons/website_sale_delivery/controllers/main.py

```diff
@@ -85,6 +85,9 @@ def process_express_checkout(self, shipping_address, shipping_option, **post):
         :param dict post: other information used in the parent route.
         :return int: The order's partner id.
         """
+        if not (shipping_address and shipping_option):
+            return super().process_express_checkout(**post)
```

Comment:
The parent method doesn't seem to accept any kwarg.

Reply:
It's already like that below. When using the route, the arguments are all name params.